build/opentsdb/start-opentsdb.sh

```diff
@@ -6,8 +6,14 @@ export COMPRESSION=NONE
 export HBASE_HOME=/opt/hbase
 export JAVA_HOME=/usr/lib/jvm/jre
 
-# Overwrite the default open tsdb config if available
 true ${OPENTSDB_RESOURCE:=/usr/local/serviced/resources/opentsdb}
+
+# Overwrite the default metric consumer config if available
+if [ -e "$OPENTSDB_RESOURCE/metric-consumer-app/configuration.yaml" ]; then
```

Comment:
If there is a chance the config file in the container will be used then it should have the new defaults from the ticket.  If it will never be used this can be merged.